ups/eupspkg.cfg.sh

```diff
@@ -44,12 +44,15 @@ install()
         # Install packages on which the stack is known to depend
 
         export PATH="$PREFIX/bin:$PATH"
-        local tmpdir=${TMPDIR:-/tmp}
         local reqfile='conda_packages.txt'
         local baseurl='https://raw.githubusercontent.com/lsst/lsstsw/master/etc/'
-        (cd "$tmpdir"; $CURL -# -L -O "${baseurl}/${reqfile}")
+        local tmpfile
+        tmpfile=$(mktemp -t "${reqfile}.XXXXXXXX")
```

Comment:
Ok. Leave it in if that is required on Linux. On OSX you just end up with a bunch of extra `X`s in the name so no problem.